lib/modules/platform/azure/index.ts

```diff
@@ -461,6 +462,22 @@ async function getMergeStrategy(
   );
 }
 
+function mapMergeStrategy(
+  mergeStrategy?: MergeStrategy,
+): GitPullRequestMergeStrategy {
+  switch (mergeStrategy) {
+    case 'rebase':
+    case 'fast-forward':
+      return GitPullRequestMergeStrategy.Rebase;
+    case 'merge-commit':
+      return GitPullRequestMergeStrategy.NoFastForward;
+    case 'squash':
+      return GitPullRequestMergeStrategy.Squash;
+    default:
+      return GitPullRequestMergeStrategy.NoFastForward;
```

Comment:
This default value is the same as here (used when `automergeStrategy` is `auto`):
https://github.com/renovatebot/renovate/blob/b8a28e0309def1ede5e723dbb460fde52e520892/lib/modules/platform/azure/azure-helper.ts#L164-L176